app/controllers/admin/settings/billing.js

```diff
@@ -0,0 +1,20 @@
+import Controller from '@ember/controller';
+import { timezones } from 'open-event-frontend/utils/dictionary/date-time';
+
+export default Controller.extend({
+  timezones,
+  updateModel() {
```

Comment:
@mrsaicharan1 I think you can use async-await here. @CosmicCoder96 am I right?

Reply:
@shreyanshdwivedi Yup.